app/src/main/java/com/android/wardrobeManager/clothing_classes/Shirt.java

```diff
@@ -0,0 +1,49 @@
+package com.android.wardrobeManager.clothing_classes;
+
+import android.os.Parcel;
+
+import com.android.wardrobeManager.ClothingItem;
+
+public class Shirt extends ClothingItem {
+    public Shirt(int color) {
+        super(color);
+    }
+
+    public static final Creator<Shirt> CREATOR = new Creator<Shirt>() {
```

Comment:
Why is this not defined in the ClothingItem class? You should try to include it there if you can because then you won't have to define it each time you create a new clothing item.